snippets/mask-arch.liquid

```diff
@@ -0,0 +1,5 @@
+<svg class="shape--mask"  width="0" height="0" viewBox="0 0 100 100">
+    <clipPath id="shape--arch" clipPathUnits="objectBoundingBox" transform="scale(0.01, 0.01)">
```

Comment:
Can we use a different `id` naming convention here? `shape--arch` is not only using a class pattern, but it is also the name of an existing class (that uses this element). 

Naming it something like `Shape-Arch` would probably help highlight that it's an `id` and therefore must be unique.

Reply:
👍 I like that. Updated in 751153f.